src/main.h

```diff
@@ -57,7 +57,7 @@ static const unsigned int DEFAULT_DESCENDANT_LIMIT = 25;
 /** Default for -limitdescendantsize, maximum kilobytes of in-mempool descendants */
 static const unsigned int DEFAULT_DESCENDANT_SIZE_LIMIT = 101;
 /** Default for -mempoolexpiry, expiration time for mempool transactions in hours */
-static const unsigned int DEFAULT_MEMPOOL_EXPIRY = 72;
+static const unsigned int DEFAULT_MEMPOOL_EXPIRY = 720;
```

Comment:
NACK :X that's too risky imho

**Double spending is a serious problem.**
Any better ideas how we could solve this?

Why has PR https://github.com/bitcoinclassic/bitcoinclassic/pull/16 been shut down without democrativc voting?  ? Did anyone delete the other pullrequest that discussed the optional removal of RBF? Why?